### PR TITLE
Users/kmkumaran/aliasesforbuildtasks

### PR DESCRIPTION
**Pull request by kmkumaran**
Add aliases to Build tasks

### CODE REVIEW

File: Tasks/AzureFileCopy/task.json [hunk 1]
```diff
@@ -58,6 +60,7 @@
         },
         {
             "name": "ConnectedServiceNameARM",
+            "aliases": ["azureSubscription"],
```
Comment (sachinma) [inline Tasks/AzureFileCopy/task.json]: how does it work when there is a conflict in alias resolution

Comment (kmkumaran) [reply Tasks/AzureFileCopy/task.json]: It fails when used more than once in the same task. I am working with @RoopeshNair to address this.

Comment (kmkumaran) [reply Tasks/AzureFileCopy/task.json]: This is addressed now. We have unique aliases.

File: Tasks/AzureAppServiceManage/task.json [hunk 2]
```diff
@@ -17,14 +17,15 @@
...
-        "Patch": 22
+        "Patch": 23
     },
     "minimumAgentVersion": "1.102.0",
     "instanceNameFormat": "$(Action): $(WebAppName)",
     "groups": [],
     "inputs": [
         {
             "name": "ConnectedServiceName",
+            "aliases": ["azureSubscription"],
```
Comment (vincentdass) [inline Tasks/AzureAppServiceManage/task.json]: Should we make aliases to be UpperCamelCase as same as Input names?

File: Tasks/AzureCLI/task.json [hunk 3]
```diff
@@ -33,6 +33,7 @@
     "inputs": [
         {
             "name": "connectedServiceNameARM",
+            "aliases": ["azureSubscription"],
```
Comment (vincentdass) [inline Tasks/AzureCLI/task.json]: Same comment as before.
Should we make aliases to be upperCamelCase / LowerCamelCase based on the other input case?

File: Tasks/AzureRmWebAppDeployment/task.json [hunk 4]
```diff
@@ -473,6 +478,7 @@
         },
         {
             "name": "XmlVariableSubstitution",
+            "aliases": ["enableXmlVariableSubstitution"],
```
Comment (vincentdass) [inline Tasks/AzureRmWebAppDeployment/task.json]: Can we also have an alias for JSON variable substitution?

File: Tasks/AzureAppServiceManage/task.json (hunk 2 above)
Comment (kmkumaran) [reply Tasks/AzureAppServiceManage/task.json]: The naming convention was discussed and settled with current form. Adding @RoopeshNair who arrived at this format.

File: Tasks/AzureCLI/task.json (hunk 3 above)
Comment (kmkumaran) [reply Tasks/AzureCLI/task.json]: Same as above.

File: Tasks/AzureRmWebAppDeployment/task.json (hunk 4 above)
Comment (kmkumaran) [reply Tasks/AzureRmWebAppDeployment/task.json]: The aliases are added only if the "name" wasn't intuitive. I believe for Json, we have JsonFiles which was good enough and hence alias is not being added. @RoopeshNair to comment if any.

**Pull request merged**